### ISSUE TITLE
Gain/loss seems swapped

### ISSUE THREAD

**Issue by TrevorHansen** [Jaychristy23/cgt-calculator-project#1]
Great effort with the calculator! 

It seems like in some cases gains and losses are swapped?

![image](https://github.com/user-attachments/assets/6208a034-2747-4cf8-949e-cb1565a5ab92)

**Comment by Jaychristy23** [comment]
Cheers! Will take a look now.

**Comment by TrevorHansen** [comment]
Yes, It's giving the correct answer now. Can you add a test case for it?

**Comment by Jaychristy23** [comment]
tried to do a bit of research, but what’s a test case? youtube isn’t great at explaining things…Sent from my iPhoneOn 26 Nov 2024, at 11:30 AM, Trevor Hansen ***@***.***> wrote:﻿
Yes, It's giving the correct answer now. Can you add a test case for it?
